fe/fe-core/src/main/java/com/starrocks/external/iceberg/cost/IcebergTableStats.java

```diff
@@ -0,0 +1,93 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.external.iceberg.cost;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Objects;
+
+import static java.lang.String.format;
+import static java.util.Collections.unmodifiableMap;
+import static java.util.Objects.requireNonNull;
+
+public class IcebergTableStats {
```

Comment:
Why not use `Statistics` class?

Reply:
this is related with IcebergColumnStats so i add a new class for iceberg. which can also align with HiveTableStats

Reply:
I think you could use `Statistics` class.  you needn‘t align with HiveTableStats. we could remove  `HiveTableStats` class later.  Add one external table，add one xxxTableStats class, which  isn't reasonable.

Reply:
thanks @kangkaisen i will change the implementation